tests/admin_views/tests.py

```diff
@@ -344,6 +344,21 @@ def test_change_list_sorting_callable(self):
             "Results of sorting on callable are out of order."
         )
 
+    def test_change_list_sorting_property(self):
+        """
+        Ensure we can sort on a list_display field that is property
+        (column 10 is property in Article model)
```

Comment:
Add trailing comma.

Reply:
I assume you mean trailing period (.) rather than comma(,)

Reply:
Yes, trailing period.